modules/Movecraft/src/main/java/net/countercraft/movecraft/craft/BaseCraft.java

```diff
@@ -32,10 +32,7 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
no `*` imports

Reply:
If I had to guess, this is intelij doing it, in which case I recommend the following
> File > Settings > Editor > Code Style > Java > Imports > "Class count to use import with '*" and "Names count to use static import with '*'"
> 
> Set both to 9999 or so.